tests/default/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -eo pipefail
+tests_failed=0
+cd terraform
+terraform init
+terraform apply --auto-approve
+VPC_ID=`terraform output -json | jq -r '.vpc_id.value'`
+subnet_count=`aws ec2 describe-subnets | jq --arg VPC_ID "$VPC_ID" '.Subnets[]| select (.VpcId==$VPC_ID)' | jq -s length`
+terraform destroy --auto-approve
+
+if [ $subnet_count -eq 3 ]
```

Comment:
We don't default to 1x public and private subnet per AZ anymore?

Reply:
I'll leave improving the module to future PRs